services/movie.py

```diff
@@ -15,19 +16,24 @@ def get_movies(
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__icontains=title)
+
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
 
 
+@transaction.atomic
 def create_movie(
     movie_title: str,
     movie_description: str,
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
Consider adding type hints for `genres_ids` and `actors_ids` parameters in `create_movie` to match the type hints used in `get_movies`.